templates/openebs-operator-2.6.0.yaml

```diff
@@ -5854,4 +5854,20 @@ spec:
                 - pgrep
                 - "ndo"
             initialDelaySeconds: 30
-            periodSeconds: 60
\ No newline at end of file
+            periodSeconds: 60
+---
+apiVersion: scheduling.k8s.io/v1
+description: Used for system critical pods that must run in the cluster, but can be
+  moved to another node if necessary.
+kind: PriorityClass
+metadata:
+  name: system-cluster-critical
+value: 900000000
+---
+apiVersion: scheduling.k8s.io/v1
+description: Used for system critical pods that must not be moved from their current
+  node.
+kind: PriorityClass
+metadata:
+  name: system-node-critical
+value: 900001000
```

Comment:
Not sure if the naming of the PriorityClass is correct? Shouldn't it be pre-fixed as in v2.5.0 ?

Reply:
In the statefulset and ds of v2.6.0 the name is like this.